api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -394,7 +310,7 @@ public CohortDefinition getPatientsWhoWereActiveByEndOfPreviousMonthB12() {
     cd.addSearch("B10", mapStraightThrough(getPatientsWhoStartedArtByEndOfPreviousMonthB10()));
     cd.addSearch("B2A", map(transferredIn, "onOrBefore=${startDate},location=${location}"));
     cd.addSearch("B3A", map(startDrugs, encounterWithCodedObsMappings));
-    cd.addSearch("B5A", map(transferredOut, encounterWithCodedObsMappings));
+    cd.addSearch("B5A", map(transferredOut, "onOrBefore=${startDate},location=${location}"));
```

Comment:
@ynurmahomed looks like this mapping will be override when calling  getPatientsTransferredOut()